tests/components/overseerr/__init__.py

```diff
@@ -0,0 +1 @@
+"""Tests for the Overseerr integration."""
```

Comment:
ideally, you'd add tests for loading and unloading the config entry

Reply:
not here of course, but in a `test_init.py`

Reply:
Added